modxcms/revolution
# Nodejs lts and other updates

### What does it do?
It is mostly about updating `node` to the latest LTS release `14.15.0` and `node-sass` up to `5.0.0` that compatible only with LTS and current releases of the nodejs.

### Why is it needed?
Some packages already deprecated or long time without updates and they may work improperly or have vulnerabilities and it should be replaced to make the build process more convenient and safe.

### How to test
Have prepared on your local machine nodejs with version `14.15.0` or higher and go to `_build/templates/default` and run several commands:
```
rm -rf node_modules
npm install
grunt build
```

Build should be done without any issues and new compiled assets should be generated.

### Related issue(s)/PR(s)
Similar changes were done in #14315
